lib/modules/databasemodule/databasemodule.py

```diff
@@ -63,59 +61,73 @@ def interpret_message(self, msg, email, db, admin, mavenlink_id) -> Dict:
                 return {
                     "command": Commands.ADMIN_LIST_USERS
                 }
+        elif commands[0] == "register": ## TODO: I saw no reason to create new module since this is user related
```

Comment:
I think I agree with this for now but leave TODO: potentially move user management to new module. I think it's good for rn but if it scales in functionality we will want to split it

Reply:
left a todo as requested